src/components/test-results/test-results.js

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ReactEcharts from "echarts-for-react";
+import CaretSvg from "../../images/icons/caret-down.svg";
+import "./test-results.styles.scss";
+import { Label } from "../label/label";
+import format from "date-fns/format";
+import fromUnixTime from "date-fns/fromUnixTime";
+
+export const TestResults = ({ personalData = {}, results = [] }) => {
+  const xAxisData = results.map(item =>
+    format(fromUnixTime(item.date), "dd.MM.yyyy / HH:mm")
+  );
+  const yAxisData = results.map(item => ({
+    value: item.temperature,
+    itemStyle: { color: item.temperature > 37 ? "red" : "blue" }
+  }));
+
+  const option = {
+    xAxis: {
+      type: "category",
+      data: xAxisData,
+      axisLine: { show: false, lineStyle: { color: "#fff" } },
+      axisLabel: {
+        textStyle: { color: "#000" }
+      },
+      splitLine: {
+        show: false
+      }
+    },
+    yAxis: {
+      type: "value",
+      axisLine: { show: true, lineStyle: { color: "#B0BEC5" } },
+      axisLabel: { show: false },
+      splitLine: {
+        show: false
+      }
+    },
+    series: [
+      {
+        data: yAxisData,
+        type: "bar",
+        barWidth: "10",
+        markLine: {
+          lineStyle: { width: 0.5, color: "#8CA0B3", style: "dashed" },
+          data: [{ name: "hightemp", yAxis: 37 }],
+          symbol: ["none", "none"]
+        }
+      }
+    ]
+  };
+
+  return (
+    <div className="container">
+      <div className="personal-data">
+        <table className="table personal-data-table">
+          <tbody>
+            <tr>
+              <th>Vârstă</th>
+              <td>{personalData.age}</td>
+            </tr>
+            <tr>
+              <th>Sexul</th>
+              <td>{personalData.gender}</td>
+            </tr>
+            <tr>
+              <th>Fumător/fumătoare</th>
+              <td>{personalData.smoking ? "Da" : "Nu"}</td>
+            </tr>
+            <tr>
+              <th>Afectiuni agravante</th>
+              <td>{personalData.comorbidities.join(", ")}</td>
+            </tr>
+            <tr>
+              <th>În izolare</th>
+              <td>{personalData.inIsolation ? "Da" : "Nu"}</td>
+            </tr>
+            <tr>
+              <th>Alte persoane in casă</th>
+              <td>{personalData.othersInHousehold ? "Da" : "Nu"}</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
```

Comment:
I think we should have the labels dynamic as well. They shouldn't be hard-coded in the component IMO.